pykube/http.py

```diff
@@ -88,6 +89,23 @@ def send(self, request, **kwargs):
 
         if "token" in config.user and config.user["token"]:
             request.headers["Authorization"] = "Bearer {}".format(config.user["token"])
+
+        elif "exec" in config.user:
```

Comment:
We should maybe also check if the `apiVersion` for the exec config is `client.authentication.k8s.io/v1alpha1`?

Reply:
i've done that. besides, I've removed the check for `command` as it's a required field in the config anyway